### ISSUE TITLE
No way to full test the whole flow for adding a specialist

### ISSUE THREAD

**Issue by danielfe** [microsoft/community-organization-operations-suite#425]
Currently we don't have a way to fully test whether adding a specialist works. You can test the admin's part of the flow (fill out the Add Specialist form and hit create) but you can't test the part where the new person gets the email and completes sign up. We need a way to get these emails so we can test that flow.

**Comment by ThomasKapler** [comment]
Actually, this does seem to work! I entered a personal email when creating a specialist and I do get an email invite from Resolve, I logged in and it does work! I can see assigned requests, notification etc.
But I agree using personal emails might not be the best. Maybe some shared test email accounts can be set up? Not sure how to do this...

**Comment by danielfe** [comment]
Yeah, I think we want to track some way to test this repeatedly even once you run through all your personal email addresses.

**Comment by ThomasKapler** [comment]
I notice that if i delete a specialist and then create a new one with the same email, it does send a new invite.

**Comment by danielfe** [comment]
Oh cool, well that’s a nice workaround, at least for now
